Sources/Public/iOS/AnimatedSwitch.swift

```diff
@@ -101,7 +104,7 @@ open class AnimatedSwitch: AnimatedControl {
     set {
       /// This is forwarded to a private variable because the animation needs to be updated without animation when set externally and with animation when set internally.
       guard _isOn != newValue else { return }
-      updateOnState(isOn: newValue, animated: false, shouldFireHaptics: false)
+      updateOnState(isOn: newValue, animated: animateUpdateWhenChangingAnimation, shouldFireHaptics: false)
```

Comment:
This is a behavior change since it previously defaulted to `false` but now defaults to `true`

Reply:
@calda This is purely a slip up! Yes I did updated in f028aca5e7d992d2b0dab1889fa7c67eadcbb313

Reply:
This is a mistake... sorry...